lerna.json

```diff
@@ -1,7 +1,6 @@
 {
   "lerna": "3.20.2",
```

Comment:
@ariellalgilmore This should be the same as the installed version of `lerna`

Reply:
fixed!